CHANGES.rst

```diff
@@ -9,6 +9,10 @@ Incompatible changes
 
 Deprecated
 ----------
+* #12630: Two ``linkcheck`` defaults are updated from deprecated legacy values:
```

Comment:
This line still reads a bit awkwardly to me.

Reply:
How about "Sphinx 8 enables two recommended linkcheck settings by default:", or a words to that effect?